CMakeLists.txt

```diff
@@ -0,0 +1,52 @@
+cmake_minimum_required(VERSION 2.8.12)
+
+project(RenderDoc CXX C)
+
+if(WIN32)
+    message(FATAL_ERROR "No Win32 support")
+endif()
+
+option(ENABLE_GL "Enable GL driver" ON)
+option(ENABLE_VULKAN "Enable Vulkan driver" ON)
+option(ENABLE_RENDERDOCCMD "Enable renderdoccmd" ON)
+option(ENABLE_QRENDERDOC "Enable qrenderdoc" ON)
+
+set(CMAKE_LIBRARY_OUTPUT_DIRECTORY "${CMAKE_BINARY_DIR}/bin")
+set(CMAKE_RUNTIME_OUTPUT_DIRECTORY "${CMAKE_BINARY_DIR}/bin")
+
+if(${CMAKE_VERSION} VERSION_GREATER 3.0)
+    set(CMAKE_CXX_STANDARD 11)
+    set(CMAKE_CXX_STANDARD_REQUIRED ON)
+else()
+    if(CMAKE_COMPILER_IS_GNUCXX)
+        set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=c++11")
+    endif()
+endif()
+
+if(CMAKE_COMPILER_IS_GNUCXX)
+    set(warning_flags
+        -Wall
+        -Wextra
```

Comment:
-Werror has been removed, I would like to keep it. If this is because of -Wextra, I would rather have either -Wextra removed or warning fixes & selective muting (as I notice there are some new -Wno... arguments). But that should probably be a separate PR.

I think at the moment, there should be no warnings with gcc and clang and the project builds with both - can you ensure that gcc/clang both build? No specific version of each, just something recent-ish.